cmd/flow-capture.go

```diff
@@ -160,140 +192,119 @@ func updateTable() {
+			}
 
-				if slices.Contains(display, pktDrop) {
-					row = append(row, []interface{}{
-						ToPacketCount(flow, "PktDropBytes"),
-						ToText(flow, "PktDropPackets"),
-						ToText(flow, "PktDropLatestState"),
-						ToText(flow, "PktDropLatestDropCause"),
-					}...)
+			// enrichment fields
+			if !slices.Contains(enrichement, noEnrichment) {
+				if slices.Contains(enrichement, zoneEnrichment) {
+					cols = append(cols,
+						"SrcZone",
+						"DstZone",
+					)
 				}
 
-				if slices.Contains(display, dns) {
-					row = append(row, []interface{}{
-						ToText(flow, "DnsId"),
-						ToDuration(flow, "DnsLatencyMs", time.Millisecond),
-						ToText(flow, "DnsRCode"),
-						ToText(flow, "DnsErrno"),
-					}...)
+				if slices.Contains(enrichement, hostEnrichment) {
+					cols = append(cols,
+						"SrcHostName",
+						"DstHostName",
+					)
 				}
 
-				if slices.Contains(display, rtt) {
-					row = append(row, []interface{}{
-						ToDuration(flow, "TimeFlowRttNs", time.Nanosecond),
-					}...)
+				if slices.Contains(enrichement, ownerEnrichment) {
+					cols = append(cols,
+						"SrcOwnerName",
+						"SrcOwnerType",
+						"DstOwnerName",
+						"DstOwnerType",
+					)
 				}
 
-				tbl.AddRow(row...)
+				if slices.Contains(enrichement, resourceEnrichment) {
+					cols = append(cols,
+						"SrcName",
+						"SrcType",
+						"DstName",
+						"DstType",
+					)
+				}
+			} else {
+				cols = append(cols,
+					"SrcAddr",
+					"SrcPort",
+					"DstAddr",
+					"DstPort",
+				)
 			}
 
-			// inserting empty row ensure minimum column sizes
-			emptyRow := []interface{}{
-				strings.Repeat("-", 16), // TimeFlowEndMs
-				strings.Repeat("-", 16), // SrcAddr
-				strings.Repeat("-", 6),  // SrcPort
-				strings.Repeat("-", 16), // DstAddr
-				strings.Repeat("-", 6),  // DstPort
-				strings.Repeat("-", 10), // FlowDirection
-				strings.Repeat("-", 16), // Interface
-				strings.Repeat("-", 6),  // Proto
-				strings.Repeat("-", 8),  // Dscp
-				strings.Repeat("-", 6),  // Bytes
-				strings.Repeat("-", 6),  // Packets
-			}
+			// standard / feature fields
+			if !slices.Contains(display, standardDisplay) {
+				if slices.Contains(display, pktDropDisplay) {
+					cols = append(cols,
+						"DropBytes",
+						"DropPackets",
+						"DropState",
+						"DropCause",
+					)
+				}
 
-			if slices.Contains(display, pktDrop) {
-				emptyRow = append(emptyRow, []interface{}{
-					strings.Repeat("-", 12), // PktDropBytes
-					strings.Repeat("-", 12), // PktDropPackets
-					strings.Repeat("-", 20), // PktDropLatestState
-					strings.Repeat("-", 40), // PktDropLatestDropCause
-				}...)
+				if slices.Contains(display, dnsDisplay) {
+					cols = append(cols,
+						"DnsId",
+						"DnsLatency",
+						"DnsRCode",
+						"DnsErrno",
+					)
+				}
+
+				if slices.Contains(display, rttDisplay) {
+					cols = append(cols,
+						"RTT",
+					)
+				}
+			} else {
+				cols = append(cols,
+					"Dir",
+					"Interface",
```

Comment:
both Dir and interface needs to be list now ?

Reply:
It will in a later PR, for now I'm using custom images since CLI PRs are hanging since a while